packages/sensor/example/lib/main.dart

```diff
@@ -0,0 +1,115 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'package:flutter/material.dart';
+import 'package:sensor/sensor.dart';
+
+import 'snake.dart';
+
+void main() {
+  runApp(new MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return new MaterialApp(
+      title: 'Sensor Demo',
+      theme: new ThemeData(
+        primarySwatch: Colors.blue,
+      ),
+      home: new MyHomePage(title: 'Flutter Demo Home Page'),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
+  final String title;
+
+  MyHomePage({Key key, this.title}) : super(key: key);
+
+  @override
+  _MyHomePageState createState() => new _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  List<double> _accelerometerValues;
+  List<double> _gyroscopeValues;
+  List<StreamSubscription<List<double>>> _streamSubscriptions =
+      <StreamSubscription<List<double>>>[];
```

Comment:
We had analysis options requiring type annotations, but I have made a PR for removing those since we have agreed on Dart conventions.